extension/inet/inet_escape_functions.cpp

```diff
@@ -0,0 +1,242 @@
+#include "duckdb/main/extension_util.hpp"
+#include "duckdb/common/types/blob.hpp"
+#include "utf8proc_wrapper.hpp"
+#include "inet_extension.hpp"
+#include "inet_functions.hpp"
+#include "html_charref.hpp"
+#include "re2/re2.h"
+
+namespace duckdb {
+
+using namespace inet;
+
+// see https://html.spec.whatwg.org/multipage/parsing.html#numeric-character-reference-end-state
+unique_ptr<unordered_map<int, string>> ReturnInvalidCharrefs() {
```

Comment:
nit: the `unique_ptr` here seems unnecessary - could we directly return an `unordered_map` instead?